comments/classified.csv

```diff
@@ -2178,15 +2117,6 @@ Time running after adding this new test,0,0.5
 "@github I would consider keeping this test as #code considering that the test above, #code, which uses a similar code minus the insane amount of dictionary instances, is",0,0.5
 already signaled to run on #code,0,0.5
 "holy crap, this is what does the LRU magic",0,0.5
-* [ ] How are we lanning on dealing with the numerous TabView quirks in v2? Just ship it as-is and fix in post? Or manually adjust the styling now,0,0.5
-* [ ] EX the close button hover is weirdly wide,0,0.5
-* [ ] the tab text is bold,0,0.5
-"* [x] Icons throughout the app are still on the Segoe MDL2, not Segoe Fluent",0,0.5
-* [x] Tab color picker looks fine,0,0.5
-* [x] Tab renamer looks fine (if maybe needs a little padding...),0,0.5
-* [x] Window Renamer / ID toast looks fine,0,0.5
-"* [x] About looks... Fine? The links aren't underlined anymore, but that might just be how WinUI works these days",0,0.5
-* [x] #codes look totally ~~insane~~ transparent,0,0.5
```

Comment:
A lot of the deleted rows seem like they were OK, can we keep these?

Reply:
Fixed!